cefi/handler/capitalcom.py

```diff
@@ -67,6 +67,7 @@ def __init__(
             logger.debug("Account data: {}", self.accounts_data)
             self.account_number = self.accounts_data["accounts"][0]["accountId"]
             logger.debug("Account number: {}", self.account_number)
+            logger.debug("Session details: {}", self.client.get_sesion_details())
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (typo):** Potential typo in 'get_sesion_details' method name.

Please verify if the method name should be 'get_session_details' instead of 'get_sesion_details'.